.github/workflows/reusable-docs.yml

```
@@ -40,12 +40,13 @@ jobs:
       uses: Ana06/get-changed-files@v2.2.0
       with:
         filter: "Doc/**"
+        format: csv  # unbreaks paths /w whitespaces; json is harder to parse
     - name: 'Build changed files in nit-picky mode'
       if: github.event_name == 'pull_request'
       continue-on-error: true
       run: |
         # Mark files the pull request modified
-        touch ${{ steps.changed_files.outputs.added_modified }}
+        python Doc/tools/touch-clean-files.py --clean ${{ steps.changed_files.outputs.added_modified }}
```

@hugovk how about this? An unquoted argument is essentially a missing arg during a CLI invocation. Plus, without erroring out on intermediate invocations, the last command could produce successful return code and lead to confusing behavior.
```suggestion
        set -Eeuo pipefail
        # Mark files the pull request modified
        python Doc/tools/touch-clean-files.py --clean '${{ steps.changed_files.outputs.added_modified }}'
```

---

Yes to quotes, and we should also modify `Doc/tools/touch-clean-files.py` so if we get `--clean ''` it should return early and do nothing at all -- don't touch the `.nitignore` files either.

For a non-zero `--clean` arg input, we touch those input CSV files.

When no `--clean` arg, we do the `.nitignore` files.